robosats/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: ${APP_ROBOSATS_IP}
+      APP_PORT: ${APP_ROBOSATS_PORT}
+
+  web:
+    image: recksato/robosats-client:7083423@sha256:35e06c9583f1efbf6765eb91d31420c5deb875e67b22443d6eb8f2ae56c5d483
```

Comment:
The digest here appears to be for `amd64`. This should be: `sha256:289f47608f2a325cbd7560d3f43f2bcb16ff57edc492ba65867c81f54d48fb38`. You can get a multi-arch digest by running `docker pull <image>` and then it will display the digest at the bottom